src/program.js

```diff
@@ -174,13 +183,16 @@ export class Program {
       }
       if (globalEnv === 'production') {
         checkForUpdates ({
-          version: getVersion(absolutePackageDir),
+          version: getVersion(this.absolutePackageDir),
```

Comment:
Can't you use the `version` variable from line 168? Like this: `checkForUpdates({version});`